catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolver.java

```diff
@@ -35,6 +39,8 @@
 
     public static final String SOURCE_ID = "source-id";
 
+    private static final String EXT_SORT_BY = "EXT_SORT_BY";
```

Comment:
`EXT_SORT_BY` is defined in a number of places. Do you think it's worthwhile to make this a constant somewhere?